### ISSUE TITLE
Modifying time signature of a given measure throws null pointer exception and generates corrupted output.

### ISSUE THREAD

**Issue by PBaciu** [kaazii/EECS2311#49]
Reporter: Patrick Alexander Baciu
Email: baciupat@my.yorku.ca
Version: 8013aee
Platform: Windows 10

Name:
Modifying time signature of a given measure throws null pointer exception and generates corrupted output.

Description:
When attempting to change the time signature of a specific measure (i.e measure range start = measure range end), a null pointer exception is thrown and the output of the music xml file is corrupted and unreadable.

Steps to Produce:
- Start the application.
- Paste in this sample guitar tab.
```
|-----------0-----|-0---------------|
|---------0---0---|-0---------------|
|-------1-------1-|-1---------------|
|-----2-----------|-2---------------|
|---2-------------|-2---------------|
|-0---------------|-0---------------|
```
- Click Translate
- Click the + to add a time signature.
- Change the dropdown to 3/4 and specify from measure 2 to 2, indicating that measure 2 should be in 3/4
- Click confirm (generates null pointer exception) [screenshot 1]
- Click save (generates corrupted xml) [screenshot 2 and screenshot3]

Expected result:
Measure 2 should be in 3/4 time and thus the duration of the chord should change.

Actual Result:
Null pointer exception + corrupted output.

Input Used:
|-----------0-----|-0---------------|
|---------0---0---|-0---------------|
|-------1-------1-|-1---------------|
|-----2-----------|-2---------------|
|---2-------------|-2---------------|
|-0---------------|-0---------------|

Priority:
Medium

Status:
Ready for development

Screenshots:
![image](https://user-images.githubusercontent.com/77293069/112344235-46772e00-8c9a-11eb-8f32-3c741a75f7df.png)
![image](https://user-images.githubusercontent.com/77293069/112344328-5bec5800-8c9a-11eb-9f74-f2f5965623d8.png)
![image](https://user-images.githubusercontent.com/77293069/112344375-66a6ed00-8c9a-11eb-8a97-3cf05a53717e.png)

**Comment by zhouziqi13** [comment]
@PBaciu Thank you for bringing this up. #41 mentions this issue. Closed

**Comment by PBaciu** [comment]
I don't think #41 mentions this issue, did you mean to close #53 ?

**Comment by zhouziqi13** [comment]
> I don't think #41 mentions this issue, did you mean to close #53 ?

Hi @PBaciu Issue mentioned in #41 is causing this error. #53 is closed for merging similar issues and features. Will be addressed in #41. 
Actually will be addressed in #36
